smart-contracts/contracts/CosmosBridge.sol

```diff
@@ -450,4 +450,42 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
     // prophecy completed and whether or not the call to bridgebank was successful
     emit LogProphecyCompleted(prophecyID, success);
   }
+
+  /**
+   * @dev Verifies if `cosmosDenom` is a ibc token
+   * @param cosmosDenom The denom of the token in sifnode
+   * @return Boolean: is `cosmosDenom` a ibc token?
+   */
+  function isIbcToken(string calldata cosmosDenom) private view returns (bool) {
+    bytes calldata cosmosDenomBytes = bytes(cosmosDenom);
```

Comment:
What about rowan?

Reply:
rowan is not ibc token, the bridge token address is available from claim.tokenAddress

Reply:
Is that going to be true on other networks we deploy to?